python/requirements.txt

```diff
@@ -16,7 +16,6 @@ PyYAML==5.4
 requests==2.24.0
 responses==0.12.0
 sinfo==0.3.1
-umap-learn==0.4.6
```

Comment:
not needed on python image

Reply:
I just saw this dependency in an unrelated context and was confused by it. I suppose it's a vestigial remnant.

Reply:
yeah there are probably other that may not be needed either but doesn't seem so important since image size will be ~100MB now